### ISSUE TITLE
Non-working `polymorphic...subclass` code sample

### ISSUE THREAD

**Issue by iirekm** [Kotlin/kotlinx.serialization#2270]
The docs say:
```
val module = SerializersModule {
    polymorphic(Any::class) {
        subclass(OwnedProject::class)
    }
}
```

It doesn't work, because `SerializersModuleBuilder` has another `polymorphic` method, and the `polymorphic` needed for this code sample an extension method, overridden by the method from `SerializersModuleBuilder`.

Version: 1.5.0

Until it's fixed, how can I workaround this?

**Comment by iirekm** [comment]
I found a workaround: use `import kotlinx.serialization.modules.polymorphic` - it's confusing but anyway.

**Comment by sandwwraith** [comment]
Yes, a lot of convenience methods are extensions, so we recommend to use star imports.
